[2024-12-03T12:42:48.047+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-03T12:42:48.075+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_to_minio.consume_kafka_to_minio manual__2024-12-03T12:42:43.367075+00:00 [queued]>
[2024-12-03T12:42:48.089+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_to_minio.consume_kafka_to_minio manual__2024-12-03T12:42:43.367075+00:00 [queued]>
[2024-12-03T12:42:48.091+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-03T12:42:48.111+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): consume_kafka_to_minio> on 2024-12-03 12:42:43.367075+00:00
[2024-12-03T12:42:48.119+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=942) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-03T12:42:48.121+0000] {standard_task_runner.py:72} INFO - Started process 944 to run task
[2024-12-03T12:42:48.122+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kafka_to_minio', 'consume_kafka_to_minio', 'manual__2024-12-03T12:42:43.367075+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag2_Kafkas3.py', '--cfg-path', '/tmp/tmp261an9bm']
[2024-12-03T12:42:48.125+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask consume_kafka_to_minio
[2024-12-03T12:42:48.196+0000] {task_command.py:467} INFO - Running <TaskInstance: kafka_to_minio.consume_kafka_to_minio manual__2024-12-03T12:42:43.367075+00:00 [running]> on host 31ac0cc6adcb
[2024-12-03T12:42:48.332+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_to_minio' AIRFLOW_CTX_TASK_ID='consume_kafka_to_minio' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T12:42:43.367075+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-03T12:42:43.367075+00:00'
[2024-12-03T12:42:48.334+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-03T12:42:49.383+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:50.386+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:51.429+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:52.430+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:53.432+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:54.433+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:55.435+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:56.437+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:57.441+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:58.445+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:42:59.447+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:00.449+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:01.450+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:02.452+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:03.453+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:04.454+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:05.473+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:06.474+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:07.476+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:08.478+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:09.481+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:10.483+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:11.485+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:12.487+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:13.489+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:14.490+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:15.493+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:16.501+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:17.503+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:18.506+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:19.507+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:20.509+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:21.524+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:22.526+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:23.528+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:24.534+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:25.535+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:26.537+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:27.538+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:28.539+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:29.541+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:30.555+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:31.558+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:32.562+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:33.565+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:34.570+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:35.595+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:36.597+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:37.599+0000] {logging_mixin.py:190} INFO - нет сообщений
[2024-12-03T12:43:38.536+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-12-03T12:43:38.568+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-03T12:43:38.572+0000] {process_utils.py:132} INFO - Sending 15 to group 944. PIDs of all processes in the group: [944]
[2024-12-03T12:43:38.578+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 944
[2024-12-03T12:43:38.608+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-03T12:43:38.739+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-03T12:43:38.826+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=944, status='terminated', exitcode=0, started='12:42:47') (944) terminated with exit code 0
